moped-data-events/activity_log/deploy_lambda.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd venv/lib/python3.8/site-packages
+zip -r9 ${OLDPWD}/lambda.zip .
+cd $OLDPWD
+zip -g lambda.zip app.py config.py MopedEvent.py
+aws lambda update-function-code --function-name atd-moped-events-activity_log_test --zip-file fileb://lambda.zip
```

Comment:
This is a small script / utility that has been useful in developing on these lambda functions. It pushes up a new package for this particular lambda function, avoiding having to do a whole fresh test deployment to heroku. 

I'm open to removing this since it's a little out of place here, but I did want to get it on the books in git in case we need to dig it up again.

Reply:
Do you want us to test this script? It might be useful to include some of this explanation in the script.

Reply:
I don't think it needs testing, it's only ever really handy to a developer working with that lambda function, but it may at least save the next developer from having to script it.